app/experimenter/experiments/constants.py

```diff
@@ -358,6 +359,62 @@ class ExperimentConstants(object):
         )
     )
 
+    ADDON_NAME_HELP_TEXT = """
+      <p>
+        Enter the name of your addon.
```

Comment:
```suggestion
        Enter the name of your add-on.
```

Perhaps it should be capitalized to "Add-On" to be most consistent.